tree/tree/src/TChain.cxx

```diff
@@ -575,6 +577,31 @@ Int_t TChain::AddFile(const char* name, Long64_t nentries /* = TTree::kMaxEntrie
    return 1;
 }
 
+
+////////////////////////////////////////////////////////////////////////////////
+/// Add a new list of files to this chain.
+///
+/// Filename formats are similar to TChain::Add. Wildcards are not
+/// applied. urls may also contain query and fragment identifiers
+/// where the tree name can be specified in the url fragment.
```

Comment:
Could you document the return value, too, please?

Reply:
done